### ISSUE TITLE
The stderror log should show calendar connection errors

### ISSUE THREAD

**Issue by spkane** [google/calblink#61]
I am currently getting a magenta blinking light on the blink(1) device, which should mean there was a problem connecting to the calendar, yet there is nothing in the error log. It would be helpful when debugging to provide a proper error in the log with some details about the underlying issue.

I know this can be seen with `-debug`, but that should not be required for such a core issue.

**Comment by kazrakcom** [comment]
Agreed - the entire logging/debug system needs an overhaul, especially with support for service mode.
